CHANGELOG.md

```diff
@@ -66,6 +70,8 @@ This release has an [MSRV][] of 1.75.
 
 - Initial release
 
+[#49][https://github.com/linebender/velato/pull/49]
```

Comment:
That's not the right syntax. There is a list of them further below, so it should just at the end of that list, with the right syntax.

Reply:
Whoops, good catch. Fixed (actually this time).